examples/call_a_tool_with_llm.py

```diff
@@ -0,0 +1,49 @@
+"""
+This example shows how to call a tool that requires authorization with an LLM using the OpenAI Python client.
+Arcade AI supports using models that are not OpenAI models, such as Anthropic's Claude. To use a non-OpenAI model,
+simple use the model name in the `model` parameter just as you would with an OpenAI model.
+Don't forget to set the model provider's API key in your arcade.env file!
+"""
```

Comment:
```suggestion
"""
This example shows how to call a tool that requires authorization with an LLM using the OpenAI Python client.
"""
```

I think we should keep the example focused on only the LLMs that are supported by Arcade Cloud Engine. Reason: we blur the lines when mentioning things like `arcade.env`, since that only applies to local installs. Ideally all the examples run out of the box against Arcade Cloud Engine. (open for discussion if you disagree of course!)